src/SumatraPDF.cpp

```diff
@@ -2001,30 +2002,18 @@ static void RerenderEverything()
 
 void GetFixedPageUiColors(COLORREF& text, COLORREF& bg)
 {
-    if (gGlobalPrefs->useSysColors) {
-        text = GetSysColor(COLOR_WINDOWTEXT);
-        bg = GetSysColor(COLOR_WINDOW);
-    }
-    else {
-        text = gGlobalPrefs->fixedPageUI.textColor;
-        bg = gGlobalPrefs->fixedPageUI.backgroundColor;
-    }
+    

```

Comment:
How do you see this toggle, as a menu item?

I don't have an opinion here. I would have to use the app with theme support for a bit to get a feel for how it works.

Also we can do a pre-release with this new code and ask people for feedback on the forum.